layouts/partials/head.html

```diff
@@ -52,7 +52,7 @@
   {{ if hasPrefix .Site.GoogleAnalytics "G-"}}
     {{ template "_internal/google_analytics.html" . }}
   {{ else }}
-    {{ template "_internal/google_analytics_async.html" . }}
+    {{ template "_internal/google_analytics.html" . }}
   {{ end }}
```

Comment:
@Okabe-Junya Thank you!

In your suggestion, L53 (`{{ template "_internal/google_analytics.html" . }}`) takes effect regardless of whether `if` in L52 is true or not.

How about this?

[Option 1: comment out redundant codes]
```
  {{/* if hasPrefix .Site.GoogleAnalytics "G-" */}}
    {{ template "_internal/google_analytics.html" . }}
  {{/* else }}
    {{ template "_internal/google_analytics_async.html" . }}
  {{ end */}}
```

[Option 2: remove redundant codes]
```
  {{ template "_internal/google_analytics.html" . }}
```

(I have not tested those code snippets. We need to check whether those codes actually works without error.)

Reply:
Thanks!

In GA4, the Async file has been deprecated, and this is now enough to configure. (https://github.com/gohugoio/hugo/issues/10899 is a useful reference)

FYI: https://gohugo.io/templates/embedded/#google-analytics

Reply:
Oh, I misunderstood the Option 2 you proposed. Option 2 is the most reasonable; I will fix

Reply:
fixed by https://github.com/cncf/glossary/pull/3210/commits/9a968122d34585740284824813b1ce6ddd9c6cdb